operator/apis/platform/v1alpha1/central_types.go

```diff
@@ -120,6 +120,11 @@ type CentralComponentSpec struct {
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,order=6,displayName="Central DB Settings (Technology Preview)"
 	DB *CentralDBSpec `json:"db,omitempty"`
 
+	// Configures telemetry settings for Central. If enabled, Central transmits telemetry and diagnostic
+	// data to a remote storage backend.
+	//+operator-sdk:csv:customresourcedefinitions:type=spec,order=7,displayName="Central telemetry settings"
```

Comment:
So, as far as I understand this:
This [API marker](https://sdk.operatorframework.io/docs/building-operators/golang/references/markers/) will render the specific option within the UI for customers to discover during installation of the CR within e.g. the OpenShift console.
Since we opted out to _officially_ document it, personally I'd say it would be best to just not show it within the UI during operator installation for now.

Might be a good question for someone from Draco: In case we have knobs in the CR we do not _yet_ want to expose, is it possible to skip this API marker?

Alternatively, we could do something vaguely similar to the central DB stuff and declare it as "Experimental" or something like that (albeit, no customer will be actually able to use it yet, so I still prefer not showing it at all).

Reply:
Agree that we should not show this during installation if possible. Asked the Draco team about this topic.

Reply:
I added a `hidden` marker to the Telemetry options. This hides the option in the UI.